### DISCUSSION TITLE
Issues with Chat GPT template

### DISCUSSION THREAD

**Discussion by amp1016** [AntlerVC/firetable#1269] [Support / Q&A] [answered]
Hi there, I have finished setting up the WhatsApp Business Support template. But I am failing to get the AI reply. 

Not sure what is missing or what part is not working. As you can see in the image upload, the reply column on the table is empty. So, Twilio is working, also Open AI has credit, and I am sure the Keys are correctly set up. Please let me know what I need to check and rectify to make the AI trigger.

![Table-•-whatsapp-ai-fe76f-•-Rowy](https://github.com/rowyio/rowy/assets/132933823/e6162588-bcd3-4eff-87c5-c583dacf4207)

**Comment by valatas37** [comment]
same problem

**Comment by BeeBombshell** [comment]
Hey @amp1016, @valatas37, the table shows that the client is offline. We're trying to debug this kind of issue but, since we are not able to replicate this, we're not sure what's the cause. Could you help us with answering the following questions?

- Do you have multiple tabs open with Rowy?
- Does the issue continue to happen if you clear your cache and log in to Rowy again?
- If so, are you able to use https://jam.dev/ chrome extension to do a quick recording, as that will help us debug this issue better.

**Reply by valatas37** [reply, marked as answer]
I made a mistake of naming secret key by other name than openai. once I recreate with openai name it works. Thanks.
